.github/workflows/build.yaml

```diff
@@ -0,0 +1,30 @@
+name: Build Docker image
+
+on:
+  push:
+    branches: ['main']
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Pin this to `ubuntu-24.04`. We've run into quite a few unexpected breakages due to the ubuntu version bumping on us.

Reply:
Done in cd731d0ebcfb83d56d07a6dec99807481e4788fb